app/src/main/res/values-es/strings.xml

```diff
@@ -252,5 +252,5 @@
 
     <!-- This is the placeholder text for an empty or null username. -->
     <string name="no_username">(no existe usuario)</string>
-    <!--<string name="sync_timed_out">La sincronización ha expirado</string>-->
+    <string name="sync_timed_out">La sincronización ha expirado</string>
```

Comment:
Is this being removed, or uncommented?

Reply:
Uncommented. We'll likely want a force-sync on the pontoon side once the string is actually removed. CC @Delphine for that.